.gitignore

```diff
@@ -7,3 +7,4 @@ pnpm-lock.yaml
 yarn.lock
 .nyc_output
 *.map
+docs/_site
```

Comment:
Correct, the GitHub Action in this PR will automatically take the entirety of `docs/_site` and push it up as a commit, so it can be published via GitHub Pages.